src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,20 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
```

Comment:
it is good practice to decompose props obj
`const { name, email, .. } = comment;`